knowledge_repo/converters/ipynb.py

```diff
@@ -46,13 +58,17 @@ def from_file(self, filename):
         c = Config()
         # c.ExtractOutputPreprocessor.extract_output_types = set()
         c.ExtractOutputPreprocessor.output_filename_template = 'images/{unique_key}_{cell_index}_{index}{extension}'
+        c.NbConvertBase.display_data_priority = ['application/javascript', 'text/html', 'text/markdown',
+                                                 'image/svg+xml', 'text/latex', 'image/png', 'image/jpeg',
+                                                 'text/plain']
 
         nb = nbformat.read(filename, as_version=4)
 
         dl = DictLoader({'full.tpl': TEMPLATE})
         md_exporter = MarkdownExporter(config=c, extra_loaders=[
                                        dl], template_file='full.tpl')
         (body, resources) = md_exporter.from_notebook_node(nb)
+        body = body.replace('</div>\n<script', '</div>\n\n<script')
```

Comment:
Cells that have `bokeh` plots are HTML cells that look like:

```html
    <div class="bk-root">
        <div class="bk-plotdiv" id="0b1810b9-8707-4646-87a4-598bfb2fbc21"></div>
    </div>
<script type="text/javascript">
...
```

The downstream processor in the knowledge repo processes the script tag
improperly in this case. Meaning that it doesn't treat the `script` tag as HTML and so the plot does not render. I found that by putting a newline between the div and the
script the script tag is treated as HTML and the plot appears. You can reproduce this problem quickly by commenting out this line and adding the bokeh notebook to a repo. You'll see that bokeh loads but that no plots are rendered in the HTML view (they do appear in the Markdown view).

A more elegant solution is probably to fix the underlying issue in the Markdown processer downstream so that `script` tags appearing afer `div` tags are still treated as HTML. I tried to do just that by modifiying the `IndentsAsCellOutputProcessor` which you introduced in #256. I didn't fully understand that code and I was unable to fix the issue. So, I punted and fixed it upstream.